enterprise_data/api/v0/views.py

```diff
@@ -76,15 +76,18 @@ def get_queryset(self):
         Returns all learner enrollment records for a given enterprise.
         """
         enterprise_id = self.kwargs['enterprise_id']
-
+        search_email = self.request.GET.get('search')
```

Comment:
better to use `query_params` instead of GET for DRF
https://www.django-rest-framework.org/api-guide/requests/#query_params